docker/tds/tds-common.env

```diff
@@ -1,3 +1,4 @@
 # Common environment properties for all services
 CONFIG_SERVICE_URL=http://configuration-service:8888
-CONFIG_SERVICE_ENABLED=true
\ No newline at end of file
+CONFIG_SERVICE_ENABLED=true
+SPRING_RABBITMQ_HOST=rabbit
```

Comment:
I've already made the changes to the filesystem resources on the QA and legacy Dev servers.